eng/pipelines/templates/jobs/cmake-generate.tests.yml

```diff
@@ -28,9 +28,13 @@ parameters:
   - name: JobName
     type: string
     default: CMakeGenerate
+  - name: TimeoutInMinutes
+    type: number
+    default: 120
 
 jobs:
 - job: ${{ parameters.JobName }}
+  timeoutInMinutes: ${{ parameters.TimeoutInMinutes }}
```

Comment:
Is this case-sensitive?

I see `timeoutInMinutes` here, but `TimeoutInMinutes` in the parameter list within the `archetype-sdk-client.yml` file.

Reply:
TimeoutInMinutes is the incoming param , timputInMinutes is the parameter for the job, being set to the incoming value. we have all incoming  params being pascal case, and the job params are all camel case. Weather case sensitive or not i did it for consistency.